Formula/openj9.rb

```diff
@@ -0,0 +1,94 @@
+class Openj9 < Formula
+  desc "High performance, scalable, Java virtual machine"
+  homepage "https://www.eclipse.org/openj9/"
+  url "https://github.com/eclipse/openj9/archive/refs/tags/openj9-0.25.0.tar.gz",
+    using: :nounzip
+  sha256 "ad1349d3f6831702ef7cbf0f91998b94a0d8d4a95267f22f659a439762dfe7f2"
+  license any_of: [
+    "EPL-2.0",
+    "Apache-2.0",
+    { "GPL-2.0-or-later" => { with: "Classpath-exception-2.0" } },
+    { "GPL-2.0-or-later" => { with: "OpenJDK-assembly-exception-1.0" } },
+  ]
+
+  livecheck do
+    url :stable
+    regex(/^openj9-(\d+(?:\.\d+)+)$/i)
+  end
+
+  keg_only :shadowed_by_macos
+
+  depends_on "autoconf" => :build
+  depends_on "bash" => :build
+  depends_on "cmake" => :build
+  depends_on "nasm" => :build if Hardware::CPU.intel?
+  depends_on "ninja" => :build
+  depends_on "openjdk" => :build
+  depends_on "pkg-config" => :build
+  depends_on xcode: :build
+
+  uses_from_macos "cups"
+  uses_from_macos "zlib"
+
+  resource "omr" do
+    url "https://github.com/eclipse/openj9-omr/archive/refs/tags/openj9-0.25.0.tar.gz"
+    sha256 "c8a2baaabf051e535ed211415a27634a2e4689f576d6d099e28ceab9ff1837c0"
+  end
+
+  resource "openj9-openjdk-jdk" do
+    url "https://github.com/ibmruntimes/openj9-openjdk-jdk16.git",
+    branch:   "v0.25.0-release",
+    revision: "0c11227a21ace48b7c978e29a347822912a6df9c"
+  end
+
+  def install
+    resource("openj9-openjdk-jdk").stage buildpath
+    resource("omr").stage buildpath/"omr"
+    mkdir "openj9"
+    system "tar", "-xf", "openj9-openj9-#{version}.tar.gz",
+      "--strip-components", "1", "-C", buildpath/"openj9"
```

Comment:
Why do you need `tar` here?

Reply:
Just for keeping the same directory layout as running the `get_source.sh` in `openj9-openjdk-jdk`.